src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    Black,
```

Comment:
In enum, all entries must be in upper case

Reply:
fixed